.github/workflows/build-nym-vpn-app.yml

```
@@ -0,0 +1,145 @@
+        type: boolean
+        default: false
+      updater:
+        description: "Enable updater (Windows only) and bump metadata (always enabled for stable releases)"
+        required: false
+        type: boolean
+        default: false
+      dev_mode:
+        description: "Enable dev mode (in-app dev menu)"
+        required: true
+        type: boolean
+        default: false
```

actionlint (1.7.12)

[error] input "dev_mode" of workflow_call event has the default value "false", but it is also required. if an input is marked as required, its default value will never be used

(events)